### DISCUSSION TITLE
Error on AddJsonTranscoding

### DISCUSSION THREAD

**Discussion by manish4638** [dotnet/aspnetcore#54152] [Q&A]
![Screenshot 2024-02-21 160221](https://github.com/dotnet/aspnetcore/assets/26702507/ee102f20-3d24-4e35-973c-a390be232265)
GOT the issue while using Microsoft.AspNetCore.Grpc.JsonTranscoding 8.0.2
but accessable in 7.0.9

**Comment by avrovv** [comment]
Did you update/verify the .NET version? The package specifies the minimum required version.
